docs/local-development.md

```diff
@@ -92,6 +92,19 @@ To get the examples how to use the admin to perform actions, refer to unit test
 - PAUSER role: `test/protocol/PauseHandlerTest.js`
 - FEE_COLLECTOR role: `test/protocol/FundsHandlerTest.js`
 
+### Upgrade facets
+To test the upgrade functionality, you first need to setup upgrader account as described in previous section.
+
+To perform the upgrade you then
+- Update some of the existing facets or create new one.
+- Update config file `scripts/config/facet-upgrade.js`:
+  - "addOrUpgrade" is the list of facets that will be upgraded or added,
+  - "remove": list of facets that will be completely removed
+  - "skip" allows you to specify methods that will be ignored during the process.
+- Update `version` in `package.json`. If the version in `package.json` matches the existing version in addresses file, you will have to explicitly confirm that you want to proceed.
+- Run `npm run upgrade-facets:local`. This will deploy new facets and make all necessary diamond cuts. It also updates the existing addresses file `addresses/31337-localhost.json` and outputs the upgrade log to the console.
```

Comment:
maybe replace example address filename with template. `addresses/<chain-id>-<environment>.json`

Reply:
I replaced that with template, but I still kept the example in since this is the expected filename if they are just following all the steps.